### DISCUSSION TITLE
https://blog.3place.cn/3place/lmyn/LMYN032.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#79] [General]
# https://blog.3place.cn/3place/lmyn/LMYN032.html

https://blog.3place.cn/3place/lmyn/LMYN032.html

**Comment by picklecai** [comment]
这样愚蠢的时刻不会来到，
不要把我们的父亲来嘲笑；
他是一个言而必信的人，
我们不会为自己把女婿挑。 （1.31.18）

因为父亲就是我们的主子，
他是我们的至高无上的神，
他有意把我们嫁给谁，
我们就会成为谁的人。’ （1.31.19）

和古代中国一样，女子婚姻，由父亲做主。

**Comment by picklecai** [comment]
听了她们说的话，
风神气得暴跳如雷，
这一位尊者冲向她们，
把她们的肢体摧毁。 （1.31.20）

这样易怒的神，和宙斯一样。也是人性。

**Comment by picklecai** [comment]
俱商婆和俱舍那婆，
阿头哩多罗阇和婆苏〔289〕。 （1.31.1）

他们都光彩辉映，英勇沉着，
都渴望去尽刹帝利的职责，
他们都奉公守法，说老实话。

这几个孩子属于刹帝利。

**Comment by picklecai** [comment]
儿子们呀！你们都要保卫，
都要促进多种多样的达磨。

是不是印度人认为：促进达磨，就是人生的意义？

**Comment by picklecai** [comment]
有五座绝妙的山峰
罗列在这土地的四周。 （1.31.6）

美妙的河苏摩竭底〔294〕，
流经摩竭陀大名鼎鼎，
看上去像是一个花环，
它位于五座山王之中。 （1.31.7）

美丽的盆地，有山有水。

**Comment by picklecai** [comment]
她们都洋溢着青春活力，
梳装打扮，美丽无比。
她们来到了后花园里，
好像雨中的闪电霹雳。 （1.31.10）

她们唱呀，跳呀，
罗摩呀！她们弹奏，
这些盛装的女子
在那里尽情地享受。 （1.31.11）

这些女子肢体柔美，
妙相在大地上超群出众。
她们来到了后花园里，
好像那云彩里的彗星。 （1.31.12）

美丽，盛装，活力。
雨中的闪电霹雳，云彩里的彗星，这两个比喻都是亮眼的。
姑娘们太美好了！

**Comment by picklecai** [comment]
’我真是爱上了你们，
愿你们都成为我的老婆；
放弃掉凡人的性格，
把那永生长寿来获得。’ （1.31.14）

神总是以为别人对自己的要求有求必应，就该人人都服从他。

**Comment by picklecai** [comment]
听了她们说的话，
风神气得暴跳如雷，
这一位尊者冲向她们，
把她们的肢体摧毁。 （1.31.20）

其实这个比喻就是狂风暴雨摧残了娇嫩的花朵。
所以，下次看到台风过境后的断枝残花，要想：这是台风太喜欢它们了呀，所以由爱生恨，反而折断了。
